### ISSUE TITLE
When editing the object type, such as the title or description, it doesn't work.

### ISSUE THREAD

**Issue by omarNaifer12** [SCENE-CE/mirador-multi-user#340]
This problem occurs with all object types, except for groups.

![MMU-Personal-MicrosoftEdge2024-12-2819-48-17-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/f7231bf4-35bd-4e7d-a98b-a2a07cd19075)

**Comment by daxid** [comment]
I cannot reproduce this one on our demo instance (taht should be up to date with the Main branch), except for the Manifest tab.

 I opened a dedicated issue: https://github.com/SCENE-CE/mirador-multi-user/issues/341 as there might be some special case for manifests. I'm not entirely sure but the title might be fetch from the iiif manifest itself (a json file) and not the database. @Drilio will comment on that.

@omarNaifer12 If you want to make some tests on a demo instance, feel free to create an account here : https://mirador-multi-user.tetras-libre.fr/

**Comment by omarNaifer12** [comment]
![metadataError](https://github.com/user-attachments/assets/a89698f4-70ea-4a0d-b772-be4d384af98b)

When I save the updated data, it throws an error saying undefined. 


![functionHandleUpdateItem](https://github.com/user-attachments/assets/1633febd-0a80-4b99-9444-9b03d8fe561b)

I think this happens because the metadataFormat is undefined, and in the function, the two variables objectTypes and selectedMetadataFormat are marked with !, meaning that if they are undefined, an error will be thrown, and the function will not execute. As a result, the second condition in updateItem is ignored, which causes this issue. I solved this by adding a simple condition. I can send the PR to check if it's correct.

**Comment by omarNaifer12** [comment]
Here is the PR. Please anyone take a look and see if it solves this problem.

**Comment by daxid** [comment]
Thanks @omarNaifer12 

@Drilio will have a look. However, with the additional info that you provide, I think the problem you encounter is related to another problem I reported yesterday. The default metadata format should be Dublin Core, but for some reason it is not present by default (see metadata related checkboxes in  #338). This would also explain why I did not reproduce on ou test instance, because I already uploaded a json metadata format file manually.
